static/app/components/replays/replayView.tsx

```diff
@@ -10,54 +9,48 @@ import ReplayPlayer from 'sentry/components/replays/replayPlayer';
 import space from 'sentry/styles/space';
 
 type Props = {
-  isFullscreen: boolean;
   toggleFullscreen: () => void;
 };
 
-function ReplayView({isFullscreen, toggleFullscreen}: Props) {
+function ReplayView({toggleFullscreen}: Props) {
   return (
     <Fragment>
       <ReplayCurrentUrl />
-      <PanelNoMargin isFullscreen={isFullscreen}>
-        <PanelHeader disablePadding noBorder>
+      <PlayerContainer>
+        <Panel>
           <ReplayPlayer />
-        </PanelHeader>
+        </Panel>
         <ScrubberMouseTracking>
           <PlayerScrubber />
         </ScrubberMouseTracking>
-      </PanelNoMargin>
-      <ReplayControllerWrapper>
-        <ReplayController toggleFullscreen={toggleFullscreen} />
-      </ReplayControllerWrapper>
+      </PlayerContainer>
+      <ReplayController toggleFullscreen={toggleFullscreen} />
     </Fragment>
   );
 }
 
-const ReplayControllerWrapper = styled(PanelBody)`
-  padding-top: ${space(1)};
-`;
```

Comment:
replaced with `gap` in index.tsx and eventReplay.tsx, so the extra dom node can go away too.